configs/paths.go

```diff
@@ -73,19 +87,32 @@ func InitPaths() error {
 		return fmt.Errorf("Failed to init bitrise paths, error: %s", err)
 	}
 
-	inputEnvstorePath, err := filepath.Abs(path.Join(BitriseWorkDirPath, "input_envstore.yml"))
+	// --- Bitrise TOOLS
+	bitriseToolsDirPth := GetBitriseToolsDirPath()
+	if err := pathutil.EnsureDirExist(bitriseToolsDirPth); err != nil {
+		return err
+	}
+	pthWithBitriseTools := os.Getenv("PATH")
+	if !strings.HasPrefix(pthWithBitriseTools, bitriseToolsDirPth) {
```

Comment:
why it should be the first part of PATH env?
now it expects be bitrise tools the first part of PATH, what if someone adds a new path to PATH after bitrise setup, it will add bitrise-tools again...

why not just cheking for contains instead of prefix?